test/e2e/features/basic.feature

```diff
@@ -22,165 +22,69 @@ Feature: Basic test
         And stdout should contain "delete"
         And stdout should contain "status"
         And stdout should contain "Flags:"
-        And stdout should contain
-            """
-            Use "crc [command] --help" for more information about a command.
-            """
 
     @darwin @linux @windows
     Scenario: CRC status
         When executing crc status command fails
-        Then stderr should contain
-            """
-            Machine does not exist. Use 'crc start' to create it
-            """
+        Then stderr should contain "Machine does not exist. Use 'crc start' to create it"
 
     @linux
-    Scenario: CRC setup on Linux
+    Scenario: CRC setup
         When executing "crc setup --check-only" fails
         Then executing crc start command fails
         And executing single crc setup command succeeds
 
-    @linux
-    Scenario: Missing CRC setup
-	Given executing "rm ~/.crc/bin/crc-driver-libvirt" succeeds
-        Then executing "crc setup --check-only" fails
-        And starting CRC with default bundle fails
-	And stderr should contain "Preflight checks failed during `crc start`, please try to run `crc setup` first in case you haven't done so yet"
-	And executing crc setup command succeeds
-
     @darwin
-    Scenario: CRC setup on Mac
+    Scenario: CRC setup
         When executing "crc setup --check-only" fails
         And executing single crc setup command succeeds
-        
+
     @windows
     Scenario: CRC setup on Windows
         When executing single crc setup command succeeds
 
     @darwin @linux @windows
-    Scenario: Request start with monitoring stack
-        When setting config property "enable-cluster-monitoring" to value "true" succeeds
-        And setting config property "memory" to value "14000" succeeds
-        Then starting CRC with default bundle fails
-        And stderr should contain "Too little memory"
-        And setting config property "memory" to value "16000" succeeds
-
-    @darwin @linux @windows
-    Scenario: CRC start
+    Scenario: CRC start usecase
+        # Request start with monitoring stack
+        * setting config property "enable-cluster-monitoring" to value "true" succeeds
+        * setting config property "memory" to value "16000" succeeds
+        Given executing crc setup command succeeds
         When starting CRC with default bundle succeeds
         Then stdout should contain "Started the OpenShift cluster"
         # Check if user can copy-paste login details for developer and kubeadmin users
-        And stdout should match "(?s)(.*)oc login -u developer https:\/\/api\.crc\.testing:6443(.*)$"
-        And stdout should match "(?s)(.*)https:\/\/console-openshift-console\.apps-crc\.testing(.*)$"
-
-    @darwin @linux @windows
-    Scenario: CRC status and disk space check
+        * stdout should match "(?s)(.*)oc login -u developer https:\/\/api\.crc\.testing:6443(.*)$"
```

Comment:
what does `*` means because some place you used `And` and some place it is replaced by `*` ?

Reply:
> Gherkin also supports using an asterisk (*) in place of any of the normal step keywords. This can be helpful when you have some steps that are effectively a list of things, so you can express it more like bullet points where otherwise the natural language of And etc might not read so elegantly.

from: https://cucumber.io/docs/gherkin/reference/#Asterisk

Reply:
Yes yes, it's just a bullet point. I use them somewhere when it's an obvious "enumeration" of things to check. But it's just for fun in a way, `And` would've worked just as well.